packages/prosess-vedtak/src/components/VedtakForm.jsx

```diff
@@ -162,40 +165,43 @@ export const VedtakForm = ({
   const mellomlagredeInformasjonsbehov = aktiverteInformasjonsbehov.map(informasjonsbehov => ({
     [informasjonsbehov.kode]: dokumentdata?.[informasjonsbehov.kode] || '',
   }));
+
+  const initialValues = Object.assign(
+    {},
+    {
+      [fieldnames.SKAL_BRUKE_OVERSTYRENDE_FRITEKST_BREV]:
+        kanKunVelge(tilgjengeligeVedtaksbrev, vedtaksbrevtype.FRITEKST) ||
+        harMellomlagretFritekstbrev(dokumentdata, vedtakVarsel) ||
+        (kanHaFritekstbrev(tilgjengeligeVedtaksbrev) &&
+          !kanHaAutomatiskVedtaksbrev(tilgjengeligeVedtaksbrev) &&
+          !harMellomLagretMedIngenBrev(dokumentdata, vedtakVarsel)),
+      [fieldnames.SKAL_HINDRE_UTSENDING_AV_BREV]:
+        kanKunVelge(tilgjengeligeVedtaksbrev, vedtaksbrevtype.INGEN) ||
+        (harMellomLagretMedIngenBrev(dokumentdata, vedtakVarsel) &&
+          !harMellomlagretFritekstbrev(dokumentdata, vedtakVarsel)),
```

Comment:
Jeg lurer på om det kanskje er nok å sjekke:
```
kanKunVelge(tilgjengeligeVedtaksbrev, vedtaksbrevtype.INGEN) ||
    (harMellomLagretMedIngenBrev(dokumentdata, vedtakVarsel)
```
Se for deg at saksbehandler skulle ha lagret fritekst, og senere ha funnet at det var riktig å heller undertrykke. Da vil kanskje ikke beslutter se at brevet er undertrykket, hvis det ligger igjen en fritekst i dokumentdata.

Det var ikke tilfelle før, siden lagreknappen aldri lagret vedtakbrevtype, men med den utvidede sjekken i `harMellomlagretFritekstbrev`, er det mer aktuelt.

Reply:
sjekket opp denne, og ser at vi sender ikke med verdiene for mellomlagret fritekstbrev dersom overstyring ikke er valgt